makim/__main__.py

```diff
@@ -93,14 +92,12 @@ def _get_args():
         makim_file = makim_file_default
 
     makim.load(makim_file)
-
     target_help = []
-
-    for group in makim.config_data['groups']:
-        for target_name, target_data in group['targets'].items():
-            target_name_qualified = f"{group['name']}.{target_name}"
+    groups = makim.config_data['groups']
+    for group in list(groups.keys()):
```

Comment:
is it necessary to convert to list in a for statement?

Reply:
It isn't, I thought it would be more explicit, gonna change it back